sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,32 @@
+$control-border-radius: 100px;
+
+$control-background-color: white;
+
+.map-control {
+  z-index: 1;
+  display: inline-block;
+  flex: 1;
+  flex-direction: column;
+  position: absolute;
+  bottom: 10%;
```

Comment:
can we use: 1. not percentages and 2. the sizing vars here? https://github.com/metrotranscom/Doorway/blob/344459c7939201ef363f95239a42f1097bc4e3f1/doorway-ui-components/src/global/tokens/sizes.scss#L25 FWIW s8 for both seems pretty good

also 3. might look better (although its not the spec) to move the buttons to the top left from the bottom left to avoid the google logo, so change `bottom` to `top`? wdyt?

Reply:
Either is fine with me but I think the controls are by default on the bottom right in maps so I imagine people are used to that.